### ISSUE TITLE
Is Dolphin language-agnostic?

### ISSUE THREAD

**Issue by bit-scientist** [bytedance/Dolphin#104]
I have tested _demo_page.py_ with a document image that has only Korean plain text. It could recognize some texts, but it also recognized some texts as random nonsensical Chinese texts such as `补贴圣圣习 哭 岂创习`, `叶引圣圣`. Is there any way to improve the detection and recognition accuracy of a specific language by fine-tuning the pre-trained model? Or maybe confine the recognition to pre-defined languages? 

I should mention that the detection and recognition accuracy is not better than EasyOCR results. It has recognized non-existing words with occasional Chinese letter combinations,  `사용号적에` instead of  `사용목적에`, `合도` instead of `습도`, etc.

P.S. The demo resulted in a complete mess for the same image by the way.

**Comment by fh2019ustc** [comment]
@bit-scientist Thank you for your interest! The majority of Dolphin's training data consists of Chinese and English documents. Fine-tuning the Dolphin model on Korean data would improve its Korean parsing capabilities. Additionally, we plan to enhance Dolphin's multilingual document parsing abilities in future updates.
